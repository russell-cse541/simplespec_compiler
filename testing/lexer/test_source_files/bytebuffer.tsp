0x[1ab859b3] 0x[2ab859b3] 0x[3ab859b3] 0x[4ab859b3] 0x[5ab859b3] 
0x[6ab859b3] 0x[7ab859b3] 0x[8ab859b3] 0x[9ab859b3]
0x[a2b3c45d6e8f9a]
0x[b1234098aa2bb3cc45dd6ee8ff9aa]
0x[c19ac9daa2bb2255662abcdef3cc45dd6ee8ff9aa]
0x[ddddd1aaabbccd34579a2bb3cc45dd6ee8ff9aa]
0x[e111eeee1aa2bb3c75748392c45dd6ee8ff9aa]
0x[ffff2231aa2bb3cc45dd6ee8ff9aa]

