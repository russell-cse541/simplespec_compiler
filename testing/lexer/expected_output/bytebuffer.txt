
0x[1ab859b3]           TagNum: 304    Line: 1     Col: 1          
0x[2ab859b3]           TagNum: 304    Line: 1     Col: 14         
0x[3ab859b3]           TagNum: 304    Line: 1     Col: 27         
0x[4ab859b3]           TagNum: 304    Line: 1     Col: 40         
0x[5ab859b3]           TagNum: 304    Line: 1     Col: 53         
0x[6ab859b3]           TagNum: 304    Line: 2     Col: 1          
0x[7ab859b3]           TagNum: 304    Line: 2     Col: 14         
0x[8ab859b3]           TagNum: 304    Line: 2     Col: 27         
0x[9ab859b3]           TagNum: 304    Line: 2     Col: 40         
0x[a2b3c45d6e8f9a]     TagNum: 304    Line: 3     Col: 1          
0x[b1234098aa2bb3cc45dd6ee8ff9aa]   TagNum: 304    Line: 4     Col: 1          
0x[c19ac9daa2bb2255662abcdef3cc45dd6ee8ff9aa]   TagNum: 304    Line: 5     Col: 1          
0x[ddddd1aaabbccd34579a2bb3cc45dd6ee8ff9aa]   TagNum: 304    Line: 6     Col: 1          
0x[e111eeee1aa2bb3c75748392c45dd6ee8ff9aa]   TagNum: 304    Line: 7     Col: 1          
0x[ffff2231aa2bb3cc45dd6ee8ff9aa]   TagNum: 304    Line: 8     Col: 1          
There were no unrecognized tokens
Token Count : 15
